ee/codegen/src/generators/sandbox-file.ts

```diff
@@ -0,0 +1,113 @@
+import { python } from "@fern-api/python-ast";
+import { AstNode } from "@fern-api/python-ast/core/AstNode";
+
+import { BasePersistedFile } from "./base-persisted-file";
+
+import type { Inputs } from "./inputs";
+import type { WorkflowFile } from "./workflow";
+
+export declare namespace SandboxFile {
+  interface Args extends BasePersistedFile.Args {
+    workflowFile: WorkflowFile;
+    inputsFile: Inputs;
+    sandboxInputs: Record<string, unknown>[];
+  }
+}
+
+export class SandboxFile extends BasePersistedFile {
```

Comment:
[blocking] Can we call this `WorkflowSandbox` everywhere that we currently say `Sandbox`? I want to disambiguate from a future of having prompt sandboxes, too.